tools/SwiftSyntax/Syntax.swift

```diff
@@ -120,6 +120,24 @@ extension Syntax {
     return data.indexInParent
   }
 
+  /// The absolute position of the starting point this node. If the first token
+  /// is with leading trivia, the position points to the start of the leading
+  /// trivia.
+  public var position: AbsolutePosition {
+    return data.position
+  }
+
+  /// The absolute position of the starting point of this node, skipping any
+  /// leading trivia attached to the first token syntax.
+  public var positionAfterSkippingLeadingTrivia: AbsolutePosition {
+    return data.positionAfterSkippingLeadingTrivia
+  }
+
+  /// The textual byte length of this node including leading and trailing trivia.
+  public var byteSize: Int {
+    return data.byteSize
```

Comment:
Could you explain the use-case in your mind?

Reply:
We can have a fuzzer for sourcekit range selection. Currently, we don't have a way to tell sourcekit to select this `SyntaxNode`.